coil-compose-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+import com.android.build.api.dsl.ManagedVirtualDevice
+
+plugins {
+    id("com.android.test")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "coil.compose.benchmark"
+    compileSdk = 33
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
+    defaultConfig {
+        minSdk = 23
+        targetSdk = 33
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        // This benchmark buildType is used for benchmarking, and should function like your
+        // release build (for example, with minification on). It"s signed with a debug key
+        // for easy local/CI testing.
+        create("benchmark") {
+            isDebuggable = true
+            signingConfig = getByName("debug").signingConfig
+            matchingFallbacks += listOf("release")
+        }
+    }
+
+    testOptions {
+        managedDevices {
+            devices {
+                create("pixel2Api31", ManagedVirtualDevice::class) {
+                    device = "Pixel 2"
+                    apiLevel = 31
+                    systemImageSource = "aosp"
+                }
+            }
+        }
+    }
+
+    targetProjectPath = ":coil-sample-compose"
```

Comment:
Yes, that's the plan eventually. I have created benchmark module for compose only for now. Name of this module is aligned with compose so once we are confident about compose let's create a separate module for the view as well.

Reply:
At the moment it looks like we're adding Compose-specific baseline rules to `coil-base` since we're benchmarking against the Compose sample. We can't assume that all `coil-base` users use Compose. Ideally that module's baseline should be optimized for views.

Reply:
This profile will reside in the `coil-compose-base` module. We will create another benchmark module for `coil-base` for view-based rules. That profile will reside in the `coil-base`.